.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+### Prerequisites
+
+- [ ] Have you written a descriptive issue title?

```

Comment:
At one point I had them like that.. can't really recall why I went back on it, because reading it again I agree with you =/ Will update